commands/pro/pro.js

```diff
@@ -2,25 +2,23 @@ function fixEscapes(str) {
   return str.replace(/[^a-z0-9|]/ig, '\\$&');
 }
 
-
 let proLangRe;
 let pros;
-const TERMS_INCLUDE = './proTerms.js';
```

Comment:
so the benefit of a variable being constant is that it can't be reassigned so it will always refer to the same string. while it's true that the string literal is immutable, changing it one place doesn't change it in the other and you lose the benefits of having a variable represent the filename